__tests__/api.ts

```diff
@@ -1,42 +1,68 @@
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+
 import { api, fetchApi } from '../src/api'
 import { mockFetch, FetchMock } from './_helper'
 
+const server = setupServer()
+
+beforeAll(() => {
+  server.listen()
+})
+
+afterEach(() => {
+  server.resetHandlers()
+})
+
+afterAll(() => {
+  server.close()
+})
+
 describe('api()', () => {
-  let fetchApi: FetchMock
+  function serverMock(url_e: string, body_e: string) {
+    server.use(
+      rest.get(url_e, (_req, res, ctx) => {
+        return res.once(ctx.status(200), ctx.body(body_e))
+      })
+    )
+  }
```

Comment:
Move this to the end of the file in the scope of the file so that you can use it everywhere in the file